### DISCUSSION TITLE
Include type helpers like ActorRefFrom in the documentation

### DISCUSSION THREAD

**Discussion by platzhersh** [statelyai/xstate#3820] [Ideas]
I find the type helpers like `ActorRefFrom` really helpful when typing my applications, however, not all of them seem to be mentioned in the documentation. (Maybe even just add it under the Recipes section?)

**Comment by davidkpiano** [comment]
We're reworking the documentation (and preparing for v5) and we will definitely include all of the type helpers in there 👍
